### PR TITLE
some text on Known Good Values from LL

### PR DESCRIPTION
**Pull request by mcr**

### CODE REVIEW

**Comment by mcr** [comment]
possible text closing #78.

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -322,7 +322,19 @@ The Verifier uses the Evidence, and any Endorsements from Endorsers,
 by applying an Evidence Appraisal Policy to assess the trustworthiness of the Attester,
 and generates Attestation Results for use by Relying Parties.  The Evidence Appraisal Policy
 might be obtained from an Endorser along with the Endorsements, or might be obtained
-via some other mechanism such as being configured in the Verifier by an administrator.
+via some other mechanism such as being configured in the Verifier by an
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:325]: Why the change in this line?

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -322,7 +322,19 @@ The Verifier uses the Evidence, and any Endorsements from Endorsers,
...
+administrator.
+
+For example, for some claims the Verifier will compare the claims in the
+Evidence to known-good-values or reference values.
+In many cases these are hashes of files, SW or memory regions.
+In other cases they might be expected SW or HW versions.
+In other cases, they may be something else.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:332]:
```suggestion
For example, for some claims the Verifier might compare the values of claims in the
Evidence to known-good values or reference values.
```

It's confusing to say that for some (arbitrary) claims, one might compare the value to something unrelated like a software version.

File: draft-ietf-rats-architecture.md [hunk 3]
```diff
@@ -322,7 +322,19 @@ The Verifier uses the Evidence, and any Endorsements from Endorsers,
...
+The actual data format and semantics of a known-good-value are specific to
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:333]:
```suggestion
The actual data format and semantics of a known-good value are specific to
```

File: draft-ietf-rats-architecture.md [hunk 4]
```diff
@@ -322,7 +322,19 @@ The Verifier uses the Evidence, and any Endorsements from Endorsers,
...
+claims and implementations.
+There is no general purpose format for them or general means for comparison defined in this architecture document.
+
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:336]:
```suggestion
Similarly, for some claims the Verifier might check the values of claims in the Evidence for membership in a set, or against a range of values, or against known-bad values such as an expiration time.
```

File: draft-ietf-rats-architecture.md [hunk 5]
```diff
@@ -322,7 +322,19 @@ The Verifier uses the Evidence, and any Endorsements from Endorsers,
...
+These known-good-values may be conveyed to the Verifier as part of an Endorsement or as part of Appraisal Policy or both as these are the two input paths to the Verifier.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:337]: This statement isn't specific to known-good values, it can apply to known bad values, ranges, sets, etc.

File: draft-ietf-rats-architecture.md [hunk 6]
```diff
@@ -322,7 +322,19 @@ The Verifier uses the Evidence, and any Endorsements from Endorsers,
...
+For example, for some claims the Verifier will compare the claims in the
+Evidence to known-good-values or reference values.
+In many cases these are hashes of files, SW or memory regions.
+In other cases they might be expected SW or HW versions.
+In other cases, they may be something else.
+The actual data format and semantics of a known-good value are specific to
+claims and implementations.
+There is no general purpose format for them or general means for comparison defined in this architecture document.
+Similarly, for some claims the Verifier might check the values of claims in the Evidence for membership in a set, or against a range of values, or against known-bad values such as an expiration time.
+These known-good-values may be conveyed to the Verifier as part of an Endorsement or as part of Appraisal Policy or both as these are the two input paths to the Verifier.
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:337]:
```suggestion
These reference values may be conveyed to the Verifier as part of an Endorsement or as part of Appraisal Policy or both as these are the two input paths to the Verifier.
```
